components/org.wso2.carbon.identity.branding.preference.resolver/src/main/java/org/wso2/carbon/identity/branding/preference/resolver/UIBrandingPreferenceResolverImpl.java

```diff
@@ -203,15 +227,144 @@ Get ancestor organization ids (including itself) of a given organization. The li
 
             // No branding found. Adding the same tenant domain to cache to avoid the resolving in the next run.
             addToCache(organizationId, currentTenantDomain, currentTenantDomain);
-            throw handleClientException(ERROR_CODE_BRANDING_PREFERENCE_NOT_EXISTS, getTenantDomain());
+            throw handleClientException(ERROR_CODE_BRANDING_PREFERENCE_NOT_EXISTS,
+                    name, ORGANIZATION_TYPE, getTenantDomain());
```

Comment:
getTenantDomain() can be replaced with currentTenantDomain

Reply:
Check other places

Reply:
Addressed with https://github.com/wso2-extensions/identity-branding-preference-management/pull/40/commits/05fa6da03cb482ecbd514d7030744011bad7a98e